data/net/minecraft/world/level/chunk/storage/RegionFileStorage.mapping

```diff
@@ -1,8 +1,12 @@
-CLASS net/minecraft/world/level/chunk/storage/RegionFileStorage
+CLASS net/minecraft/world/level/chunk/storage/RegionFileStorage net/minecraft/world/level/chunk/storage/RegionFileStorage
+	COMMENT Handles reading and writing the {@link net.minecraft.world.level.chunk.storage.RegionFile region files} for a {@link net.minecraft.world.level.Level}.
+	METHOD <init> (Ljava/io/File;Z)V
+		ARG 1 regionFolder
```

Comment:
Note the brought-up point of 'parameters follow field names': `regionFolder` -> `folder`.